WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.kt

```diff
@@ -216,6 +216,10 @@ class MeFragment : Fragment(R.layout.me_fragment), OnScrollToTopListener {
 
         if (BuildConfig.IS_JETPACK_APP) meAboutIcon.setImageResource(R.drawable.ic_jetpack_logo_white_24dp)
 
+        rowExperimentalFeaturesSettings.setOnClickListener {
+            ActivityLauncher.viewExperimentalFeatures(activity)
```

Comment:
The app is failing to build because `activity` is nullable. I recommend using `requireActivity()` instead.

Reply:
Addressed in 637e41e0770ec926e35b935faf3326d2e8476c49 . 

The use of `activity` was oversight that came to light when I addressed a nullable lint warning in 4ac3e191248840a9b17b9f165cf83eee245f9ca9. This was intended to be `context`.